iosHyperskillApp/iosHyperskillApp/Sources/Frameworks/sharedSwift/Extensions/AppFeatureStateKsExtensions.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
+import shared
+
+#warning("TODO: Replce with source code generator Sourcery")
+extension AppFeatureStateKs: Equatable {
+    public static func == (lhs: AppFeatureStateKs, rhs: AppFeatureStateKs) -> Bool {
```

Comment:
1. This extension with `Equatable` conformance is needed for overriding `FeatureViewModel.shouldNotifyStateDidChange(oldState:, newState:)`, with this we achieved performance improvements because now we notify View about State updates only when it changed.
2. In the future we may discover https://github.com/krzysztofzablocki/Sourcery and try to use it for generating these extensions.